### ISSUE TITLE
Add link to Claim Status Tool page sidebar

### ISSUE THREAD

**Issue by vhenry07** [department-of-veterans-affairs/va.gov-team#28832]
## Issue Description
OmniChannel chat service request from Stephen James based on [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/28360#issuecomment-893789185).

---
## Tasks
- [ ] Add a link to the Claims and Appeals tool page for our new OmniChannel chat service. The link can go in the Needs Help section of the right sidebar.  It will need to have javascript included as well as it will launch a small popup chat window.  (chat service scheduled go live in November)

## Acceptance Criteria
- [ ] Link added to sidebar
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by vhenry07** [comment]
Emailed Stephen James for the required link info.

**Comment by Mottie** [comment]
Hi @stephenjames1,

From looking at the HTML, I think we'll need a content & accessibility review:
- I believe a button vs a link should be used to open the new window
- "Click Here" is not appropriate language for accessibility
- Exclamation points should not be used per our design style guide - https://design.va.gov/content-style-guide/punctuation#exclamation-points
- The window title of "xtf" should be better named for screen reader users
- A fixed window size of 400x600 would not be appropriate for mobile users

**Comment by Mottie** [comment]
We're still waiting on IA & Content feedback for this issue.

**Comment by Mottie** [comment]
Pushing this work to the next sprint. We're still waiting on IA & content review. See https://github.com/department-of-veterans-affairs/va.gov-team/issues/31707

**Comment by Mottie** [comment]
@steveweinberger This work has been merged into staging. It will be available in production, behind a feature flag, later today.

To add the authorized emails to the feature flags
- First add your ID.me email to https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/settings.yml#L769 (`flipper > admin_user_emails`) to allow modifying feature flags, then
- Log into that user and go to https://api.va.gov/flipper/features/omni_channel_link to add/remove users

Docs are here: https://depo-platform-documentation.scrollhelp.site/developer-docs/Feature-toggles.1859780873.html#Featuretoggles-Enablinganddisablingfeatures
